tests/client/crud_spec/framework.rs

```diff
@@ -170,7 +170,7 @@ macro_rules! run_insert_one_test {
 macro_rules! run_replace_one_test {
     ( $db:expr, $coll:expr, $filter:expr, $replacement:expr, $upsert:expr,
         $outcome:expr ) => {{
-            let options = ReplaceOptions::new($upsert, None);
+            let options = ReplaceOptions { upsert: $upsert, write_concern: None };
```

Comment:
I ran into a weird thing here when I tried to use the `..Default::default()` syntax; the compiler seemed to be treating the `..` as part of a range expression. Strangely, it only seemed to occur in certain places in macro definitions.